src/openrct2/localisation/UTF8.cpp

```diff
@@ -92,6 +92,21 @@ int32_t UTF8Length(const utf8* text)
     return count;
 }
 
+/**
+ * Gets the number of bytes required to store the first x characters / codepoints in a UTF-8 string
+ */
+size_t UTF8Limit(const utf8* text, size_t limit)
```

Comment:
I've removed this function and written a new function in `String.cpp`  which does the same thing but with string views. I moved it because UTF8.cpp seems to only have unsafe functions like this, whereas `String.cpp` has safer ones which use string views.